avm/res/event-grid/domain/event-subscription/README.md

```diff
@@ -0,0 +1,134 @@
+# Event Grid Domain Event Subscriptions `[Microsoft.EventGrid/domains/eventSubscriptions]`
+
+This module deploys an Event Grid Domain Event Subscription.
+
+## Navigation
+
+- [Resource Types](#Resource-Types)
+- [Parameters](#Parameters)
+- [Outputs](#Outputs)
+
+## Resource Types
+
+| Resource Type | API Version |
+| :-- | :-- |
+| `Microsoft.EventGrid/domains/eventSubscriptions` | [2025-02-15](https://learn.microsoft.com/en-us/azure/templates/Microsoft.EventGrid/2025-02-15/domains/eventSubscriptions) |
+
+## Parameters
+
+**Required parameters**
+
+| Parameter | Type | Description |
+| :-- | :-- | :-- |
+| [`destination`](#parameter-destination) | object | The destination for the event subscription. See EventSubscriptionDestination objects for full shape. |
+| [`domainName`](#parameter-domainname) | string | Name of the Event Grid Domain. |
```

Comment:
Should be outdated. Please regenerate the json & readme files for the module, then we can merge :)

Reply:
done

Reply:
Hey @fabmas, it may appear the change was not pushed. Please double-check :)